### PR TITLE
Increase search border highlights

### PR DESCRIPTION
**Pull request by kingthorin**
Related to: zaproxy/zaproxy#8132

Before:
image

After:
image

### CODE REVIEW

**Review by psiinon — APPROVED** [review]

**Comment by kingthorin** [comment]
Could I get a second review on this?

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**
